Utilities/CreateVault/sign.command

```diff
@@ -45,6 +50,8 @@ if [ ! -x ./RsaTool ] || [ ! -x ./create_vault.sh ]; then
   fi
 fi
 
+trap cleanup EXIT ERR INT TERM
```

Comment:
Do you believe it is better to directly calling `cleanup` in e.g. `abort`? I personally do not mind.